rbac/internal/urls.py

```diff
@@ -78,6 +78,7 @@
     path("api/utils/get_org_admin/<int:org_or_account>/", views.get_org_admin),
     path("api/utils/role/", views.role_removal),
     path("api/utils/permission/", views.permission_removal),
+    path("api/utils/role_migration/", views.role_migration),
```

Comment:
can you put it please in into internal open api spec ? 🙏

Reply:
Oh, good point! Thanks! Added now